src/ResourceManager/IotHub/Commands.IotHub/help/Add-AzureRmIotHubCertificate.md

```diff
@@ -0,0 +1,176 @@
+---
```

Comment:
Was this help pulled from the python one? Just want to make sure they're the same.

Reply:
Yes, it is almost same because in PS we have same command for create and update the certificate instead of two different commands in PY.